### PR TITLE
civo terraform cluster bug fix

### PR DESCRIPTION
**Pull request by jarededwards**
## Description
fixes missing required `write_kubeconfig` property from civo terraform provider upgrade

## Related Issue(s)
Fixes https://github.com/kubefirst/kubefirst/issues/2262 

## How to test
```
kubefirst civo create  \
  --gitops-template-branch civo-remote-ws
```

### CODE REVIEW

File: civo-github/terraform/civo/modules/workload-cluster/main.tf [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+
+resource "kubernetes_namespace_v1" "external_secrets_operator" {
+  metadata {
+    name = "external-secrets-operator"
+  }
+}
+
+resource "kubernetes_namespace_v1" "environment" {
+  metadata {
+    name = "<WORKLOAD_CLUSTER_NAME>"
```
Comment (jarededwards) [inline civo-github/terraform/civo/modules/workload-cluster/main.tf:131]:
```suggestion
    name = var.cluster_name
```

File: civo-gitlab/terraform/civo/modules/workload-cluster/main.tf [hunk 2]
```diff
@@ -0,0 +1,209 @@
...
+}
+
+resource "civo_kubernetes_cluster" "kubefirst" {
+  name             = var.cluster_name
+  network_id       = civo_network.kubefirst.id
+  firewall_id      = civo_firewall.kubefirst.id
+  write_kubeconfig = true
+  pools {
+    label      = var.cluster_name
+    size       = var.instance_size
```
Comment (jarededwards) [inline civo-gitlab/terraform/civo/modules/workload-cluster/main.tf:18]:
```suggestion
    size       = var.node_type
```

File: civo-gitlab/terraform/civo/modules/workload-cluster/main.tf [hunk 3]
```diff
@@ -0,0 +1,209 @@
...
+
+resource "kubernetes_namespace_v1" "external_secrets_operator" {
+  metadata {
+    name = "external-secrets-operator"
+  }
+}
+
+resource "kubernetes_namespace_v1" "environment" {
+  metadata {
+    name = "<WORKLOAD_CLUSTER_NAME>"
```
Comment (jarededwards) [inline civo-gitlab/terraform/civo/modules/workload-cluster/main.tf:131]:
```suggestion
    name = var.cluster_name
```

**Comment by jarededwards** [comment]
image
successfully synced workload cluster

**Review by mrsimonemms — APPROVED** [review]
Works beautifully

**Pull request merged**